docs/source/configuration/apollo-telemetry.mdx

```diff
@@ -42,6 +42,18 @@ Note that `field_level_instrumentation_sampler` may not sample at a greater rate
 
 ## Advanced configuration
 
+### `send_headers`
+
+Provide this field to configure which request header names and values are included in trace data that's sent to Apollo Studio. Valid options are: `only` with an array, `except` with an array, `none`, `all`.
+
+The default value is `none``, which means no header names or values are sent to Studio. This is a security measure to prevent sensitive data from potentially reaching the Router.
```

Comment:
This will result in broken rendering right now.  (There's a good Markdown Preview feature in VS Code that can help catch this. :wink:)

Also, I think you meant "reaching Studio", but I have a suggestion there, too.

```suggestion
The default value is `none`, which means no header names or values are sent to Studio. This is a security measure to prevent sensitive data from leaving your infrastructure unless you specifically opt into it
```